### ISSUE TITLE
Traefik - Permission denied for version 1.0.1-1

### ISSUE THREAD

**Issue by claudyus** [rancher/community-catalog#270]
Hi all,
I'm testing the træfiĸ image from `rawmind/alpine-traefik:1.0.1-1` and I get the following error from log:

```
9/8/2016 11:04:11 AM'traefik' process is not running
9/8/2016 11:04:11 AM'traefik' trying to restart
9/8/2016 11:04:11 AM'traefik' start: /opt/traefik/bin/traefik-service.sh
9/8/2016 11:04:41 AM'traefik' failed to start (exit status 0) -- /opt/traefik/bin/traefik-service.sh: time="2016-09-08T09:04:11Z" level=fatal msg="Error opening fileopen /opt/traefik/log/traefik.log: permission denied"
9/8/2016 11:04:41 AM
9/8/2016 11:05:41 AM'traefik' process is not running
9/8/2016 11:05:41 AM'traefik' trying to restart
9/8/2016 11:05:41 AM'traefik' start: /opt/traefik/bin/traefik-service.sh
9/8/2016 11:06:12 AM'traefik' failed to start (exit status 0) -- /opt/traefik/bin/traefik-service.sh: time="2016-09-08T09:05:41Z" level=fatal msg="Error opening fileopen /opt/traefik/log/traefik.log: permission denied" 
9/8/2016 11:06:12 AM
9/8/2016 11:07:12 AM'traefik' process is not running
9/8/2016 11:07:12 AM'traefik' trying to restart
9/8/2016 11:07:12 AM'traefik' start: /opt/traefik/bin/traefik-service.sh
9/8/2016 11:07:42 AM'traefik' failed to start (exit status 0) -- /opt/traefik/bin/traefik-service.sh: time="2016-09-08T09:07:12Z" level=fatal msg="Error opening fileopen /opt/traefik/log/traefik.log: permission denied"
9/8/2016 11:07:42 AM
```

If I execute a shell inside the container this is the shell (and I believe the user)  that is used also to run the init script:

```
$ id
uid=10001(traefix) gid=10001(traefix) groups=10001(traefix) 
```

I  will try to fork this community repo and upgrade composer image reference to 1.0.2-6 that is available upstream anyway I feel that this could be not a solution.

**Comment by johnrengelman** [comment]
Also seeing this...huge pain.
Wasn't able to update to 1.0.2-6 because it appears that there are major changes to the associate conf container for that.

**Comment by rawmind0** [comment]
Hi guys,

with 1.0.2-6 version there is a major changes. You could see current docker and rancher compose yml's under https://github.com/rawmind0/alpine-traefik/tree/master/rancher 

Otherwise, you could get last rancher catalog package version, adding https://github.com/rawmind0/service-catalog to your rancher catalog repos.

I'm preparing PR's to this repo.

Best regards...

**Comment by claudyus** [comment]
Ehi @rawmind0, I just deploy a traefik istance from your catalog! Sweet

Should we close the issue or it is better to keep it opened waiting the ramind0's PR?

**Comment by rawmind0** [comment]
PR added #300  ... :)

**Comment by rawmind0** [comment]
Merged PR #300... Closing Issue....
